templates/patterns/card_v2/card_v2.ui_patterns.yml

```diff
@@ -0,0 +1,64 @@
+card_v2:
+  label: 'Card V2'
+  description: 'A card is a flexible and extensible content container. The V2 leaves more possibility for content and render items as we transformed all the content areas into render options. https://v5.getbootstrap.com/docs/5.0/components/card'
+  variants:
+    default:
+      label: 'Default'
+      description: 'Standard bootstrap card v2.'
+  fields:
+    title:
+      type: text
+      label: 'Title'
+      description: 'Card title. Supports markup, e.g. for links.'
+      preview: 'Title card'
+    title_tag:
+      type: text
+      label: 'Title tag'
+      description: 'Card title tag. Default h1.'
+      preview: 'h4'
+    image:
+      type: render
+      label: 'Image'
+      description: 'Image for the card. Optional.'
+      preview: '<img src="https://picsum.photos/1000/500/" alt="Alternative text for card image">'
+    image_position:
+      type: text
+      label: 'Image position'
+      description: 'Image position of the card. Options: bottom, top. Optional. Default: top.'
+      preview: 'top'
+    card_body:
+      type: render
+      label: 'Card body'
+      description: 'Card body content.'
+      preview: '<h6 class="card-subtitle mb-2">Subtitle card</h6><div class="card-text mb-2">Text card with longer text which is supported</div><div class="my-3 me-n3"><span class="text-muted me-3">Organisation</span><span class="text-muted me-3">Position</span></div><div><button class="btn btn-primary">Custom action</button></div><div class="mt-2-5"><span class="mb-2 me-2 badge rounded-pill badge-outline-primary">A badge</span><span class="mb-2 me-2 badge badge-outline-primary">A non-rounded badge</span><span class="mb-2 badge rounded-pill text-bg-primary">A solid badge</span></div>'
```

Comment:
I forgot to write this in the previous review: we should remove most of this in here, and add a few examples of cards in the dedicated template pattern-card-v2--preview.html.twig.
This way we can show some more complex scenarios, and with time we can use other patterns directly instead of hardcoding the html.